src/security/library.cpp

```diff
@@ -0,0 +1,585 @@
+#include "library.h"
+
+#include <string>
+
+extern "C" {
+#include <ngx_core.h>
+#include <ngx_cycle.h>
+#include <ngx_log.h>
+#include <ngx_string.h>
+}
+
+#include <ddwaf.h>
+#include <rapidjson/schema.h>
+
+#include <fstream>
+#include <numeric>
+#include <optional>
+#include <stdexcept>
+#include <string_view>
+#include <utility>
+
+#include "blocking.h"
+#include "context.h"
+#include "ddwaf_obj.h"
+#include "util.h"
+
+extern "C" {
+#define INCBIN_SILENCE_BITCODE_WARNING
+#include "incbin.h"
+}
+
+extern "C" {
+INCBIN(char, RecommendedJson, "security/recommended.json");
+}
+
+using namespace std::literals;
+namespace dnsec = datadog::nginx::security;
+
+namespace {
+
+static constexpr auto kDefaultKeyRegex =
+    "(?i)(p(ass)?w(or)?d|pass(_?phrase)?|secret|(api_?|private_?|public_?)key)|"
+    "token|consumer_?(id|key|secret)|sign(ed|ature)|bearer|authorization"sv;
+
+static constexpr ddwaf_config kBaseWafConfig{
+    .limits =
+        {
+            .max_container_size = 150,
+            .max_container_depth = 10,
```

Comment:
Defaults are typically 256 and 20 respectively, why did you choose these?

Reply:
changed to those